lib/claper_web/live/event_live/manager_settings_component.ex

```diff
@@ -5,7 +5,7 @@ defmodule ClaperWeb.EventLive.ManagerSettingsComponent do
     assigns = assigns |> assign_new(:show_shortcut, fn -> true end)
 
     ~H"""
-    <div class="grid grid-cols-1 @md:grid-cols-2 @md:space-x-5 px-5 py-3">
+    <div class="grid grid-cols-1 @md:grid-cols-2 @md:space-x-5 px-5 py-3 h-full mb-10 ">
```

Comment:
```suggestion
    <div class="grid grid-cols-1 @md:grid-cols-2 @md:space-x-5 px-5 py-3 h-full mb-10">
```

You added a trailing space here ^^

Reply:
I have run `mix format`,   can you check it again  , thank you !

Reply:
Those diffs only appear in GitHub, but in VS Code, it's showing only 6-8 line changes   
![Uploading image.png…]()

Reply:
Yeah, that makes sense, seeing as the indenting has changed. You can toggle that setting in GitHub to minimise such changes 
<img width="1461" height="966" alt="image" src="https://github.com/user-attachments/assets/7cc1bd9a-746a-4de5-8256-ff77a1c1e9e2" />